rootfs/api/models/app.py

```diff
@@ -541,6 +542,7 @@ def verify_application_health(self, **kwargs):
             level=logging.DEBUG
         )
 
+    @backoff.on_exception(backoff.expo, ServiceUnavailable, max_tries=3)
```

Comment:
i think you can use @retry(retry_on_exception= ServiceUnavailable, stop_max_attempt_number=3)